data/release.py

```diff
@@ -121,16 +121,18 @@ def state(self):
   def state(self, value):
     """Sets state as State."""
     if self._validate_type(int, value, 'state'):
-      if value == 0:
+      if value == 1:
```

Comment:
Can we replace this by if 1 <= value <=6: state = value ?

Reply:
Done